### DISCUSSION TITLE
i3 Appimage

### DISCUSSION THREAD

**Discussion by HaressO** [i3/i3#6107] [Q&A]
How to access appimage programs through dmenu? I tried symlink, appimagelauncher. Also i notice that i was not be able to execute one program before i executed it in Terminal, but it doesn`t work with appimage.  (Mint - xfce)

**Comment by EvertonCosmo** [comment]
Hi @HaressO

I have put  this on dmenu_run 

```shell 
export PATH=$PATH:/var/lib/flatpak/exports/bin
export PATH=$PATH:/$HOME/AppImages 
```
